### PR TITLE
data: Salmanakhtar57

### PR DESCRIPTION
**Pull request by salmanakhtar57**

### CODE REVIEW

File: data/salmanakhtar57.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
+{
```
Comment (EmmaDawsonDev) [inline data/salmanakhtar57.json:1]: You have duplicated the first section. Please Decide which one you want to keep and remove the extra.

**Review by EmmaDawsonDev — DISMISSED** [review]

File: data/salmanakhtar57.json (hunk 1 above)
Comment (salmanakhtar57) [reply data/salmanakhtar57.json:1]: I want to keep the second one and remove the first one.

Comment (kumarsonsoff3) [reply data/salmanakhtar57.json:1]: You can remove the other extra one [here](https://github.com/salmanakhtar57/LinkFree/edit/salmanakhtar57-patch-1/data/salmanakhtar57.json) then. Feel free to let us know if you need any help with this! :)

Comment (salmanakhtar57) [reply data/salmanakhtar57.json:1]: Thanks for letting me know. I removed the extra one and commit changes. Let me know if anything else that I need to do.

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good, Thanks! 🎉

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Pull request merged**